gosuper/gosuper/api.go

```diff
@@ -236,6 +237,15 @@ func LogToDisplayControl(writer http.ResponseWriter, request *http.Request) {
 		return
 	}
 
+	if loaded, err := systemd.Dbus.GetUnitProperty(serviceName, "LoadState"); err != nil {
+		sendError(fmt.Errorf("Unable to get log to display load status: %v", err))
+		return
+	} else if loaded.Value.String() == `"not-found"` {
```

Comment:
We could use systremd's GetUnit dbus method, but go-systemd doesn't seem to support it. And we already check against a string for "active" below, so in any case this is consistent (though I agree the hardcoded strings are ugly).
BTW, I still haven't tested if this works like this, I'll check it now (it's an educated guess from what `systemctl status` returns).